lib/shared/sdk/index.js

```diff
@@ -231,7 +230,10 @@ SDK.Scanner = class Scanner extends EventEmitter {
       return this
     }
 
-    this.adapters.delete(instance.name)
+    instance.removeListener('devices', this.onDevices)
+    instance.removeListener('error', this.onError)
+
+    this.adapters.delete(instance.id)
     this.emit('unsubsribe', adapter)
```

Comment:
`unsubsribe`?? Does this PR need to be rebased?

Reply:
The typo-correction PR isn't merged yet, will rebase whichever one comes after the other

Reply:
Aha. I've just merged #1832 for you :)

Reply:
Sweet, rebased that now